CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+## Making changes to CitGM
+#### There are a few basic requirements for making changes to CitGM
+
+* Create an issue prior to submitting a PR as this can help speed up the process
+* Include tests for your code wherever possible
+* Include any documentation or manual changes where necessary
+* Ensure that `npm test` passes before submitting the PR
+* Please squash each logical change into a single commit
+* Follow the nodejs commit guidelines shown [here](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#step-3-commit)
```

Comment:
So yeah I agree with @gibfahn, I personally feel that on a smaller scale project like this, we don't perhaps need to include `Reviewed-by:` but I can see that including `PR-URL:` may be useful.